js/example.js

```diff
@@ -13,6 +13,10 @@ module.exports = function (styleguide, filePath) {
   const buildPath = styleguide.path.build()
   let data = resolver.data(buildPath, filePath)
 
+  if (Object.keys(data).length === 0) {
```

Comment:
That would be caught by the try/catch block you have in `task/ui.js` though, right?